web/js/waste.js

```diff
@@ -0,0 +1,6 @@
+$(function() {
+    $('form.waste').on('submit', function(e) {
+        $('input[type=submit]', this).prop("disabled", true);
+        $('#form-submit-row').addClass('loading');
```

Comment:
There's quite a few other button classes used here that might as well have this give we are disabling the button, e.g form-go-row on the address lookup page and also the continue buttons on the enquiry form.

Reply:
Yeah, maybe `$('input[type="submit"], this).parents('.govuk-form-group').addClass('loading')` ?